frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -191,4 +191,53 @@ describe("scenarios > question > filter", () => {
       cy.findAllByText("Fantastic Wool Shirt").should("not.exist");
     });
   });
+
+  it.skip("should filter using Custom Expression from aggregated results (metabase#12839)", () => {
+    const CE_NAME = "Last Year";
+
+    withSampleDataset(({ PRODUCTS }) => {
+      cy.request("POST", "/api/card", {
+        name: "12839",
+        dataset_query: {
+          database: 1,
+          query: {
+            filter: [">", ["field-literal", CE_NAME, "type/Float"], 0],
+            "source-query": {
+              aggregation: [
+                ["avg", ["field-id", PRODUCTS.PRICE]],
+                [
+                  "aggregation-options",
+                  [
+                    "count-where",
+                    [
+                      "time-interval",
+                      ["field-id", PRODUCTS.CREATED_AT],
+                      -1,
+                      "year",
+                    ],
+                  ],
+                  { "display-name": CE_NAME },
+                ],
+              ],
```

Comment:
Perhaps I was having too much fun, when I created steps-to-reproduce - let's simplify it and make sure it also works in 13 months
```suggestion
              aggregation: [
                [
                  "aggregation-options",
                  ["+", 1, 1],
                  { "display-name": CE_NAME }
                ]
              ],
```

Reply:
I'll commit this, but also change the `display-name` of this Custom Expression.